### DISCUSSION TITLE
proeven bekijken

### DISCUSSION THREAD

**Discussion by michelwalgraeve** [Nelos-VZW/dives-2#60] [Q&A]
wij schrijven na elke proef commentaar vooral als het niet gelukt is. Je kan in de brevetkaart van kandidaat zien dat het de vorige keer in  opleiding was afgetekend maar je kan niet de commentaar van de instructeur vinden die de proef in opleiding heeft gedaan. Ook niet of het al 1 of twee keer is geprobeerd. 
Is dat iets wat nog gaat komen of niet  want wordt in onze club veel gebruikt om info over proef en kandidaat te lezen. Je krijgt ook direct beeld van de reeds afgenomen proeven of deze goed waren of met veel moeite.

**Comment by ndsmyter** [comment]
Je kan als instructeur nog steeds een melding toevoegen als je dat wil. Daar was eigenlijk heel wat discussie over tijdens de functionele analyses. Veel mensen die dit wel gebruiken, maar evenveel mensen die dit totaal niet gebruiken.
Om iedereen tevreden te houden, zit het er in, maar is het nog steeds optioneel.

<img width="315" alt="image" src="https://user-images.githubusercontent.com/648669/190174688-b9581d74-0a16-4504-a37b-fc31b13e1492.png">
<img width="590" alt="image" src="https://user-images.githubusercontent.com/648669/190174796-fc154d15-aa3c-4c44-9a31-973b07f120d9.png">

**Reply by michelwalgraeve** [reply]
de vraag is kan je dit als andere instructeur (dus niet degene die de comment gepost heeft) dan lezen. In dives 1 gaan we  naar brevetkaart en kunnen we de vorige proeven zien met al de comments. Nu kan ik wel een comment ingeven maar kan ik die niet lezen in de brevetkaart van de kandidaat

**Reply by ndsmyter** [reply]
Als "andere instructeur" kan je dat niet lezen. Dat kunnen we nu niet meer omdat we de brevetkaarten overal "publiek" gemaakt hebben en we dan hierdoor toch mogelijks iets persoonlijkere informatie delen. Want wat er ingevuld wordt in die tekst, is meestal wat er niet zo goed ging, dat wil je liefst niet aan de grote klok hangen.

We hebben hier 2 zaken die tegen elkaar inwerken (publieker maken van brevetkaarten versus bekijken ven opmerkingen bij proeven). Aangezien er heel wat instructeurs zijn die deze opmerkingen niet gebruiken, hebben we de voorkeur gegeven om de brevetkaarten publieker te maken en de opmerkingen minder publiek.
De opmerkingen kunnen wel gelezen worden door de opleidingsverantwoordelijke en duikschoolverantwoordelijke, moest dat nodig zijn.

Ikzelf heb dat veld echt nog nooit gebruikt in DIVES 1.0. Ik doe nochtans redelijk veel proeven op een jaar, ook met mensen die ik van haar noch pluimen ken :-)
Ik praat meestal gewoon eens met de mensen en maak hen duidelijk dat als ze me zeggen wat er vorige keer mis ging, ik nog tips kan geven om het nu beter te doen. Dan wordt er meestal redelijk snel open kaart gespeeld. En als het dan toch niet goed zou zijn tijdens de proef zelf omwille van dezelfde reden, dan is dat jammer voor hen dat ze het me niet op voorhand hebben gezegd.

Moest je suggesties hebben om die meldingen wel te tonen, dan hoor ik dat graag. Maar denk dat die "publiekere brevetkaarten" nu wel eventjes zullen blijven in DIVES 2.0.

**Reply by michelwalgraeve** [reply]
bij ons in de club wordt dit echt als een nuttige tool gezien. Iedereen die coach is kan dit dan zien. Kan dit niet bijvoorbeeld voor AI en instructeurs   die van dezelfde club zijn worden opengezet.

**Reply by ndsmyter** [reply]
Goede suggestie.
Ik neem het mee om dit te bespreken. Dit is niet zomaar iets dat ik zelf de beslissing over wil nemen.
Ik heb ondertussen wel al #67 aangemaakt hiervoor.

**Comment by michelwalgraeve** [comment]
kan je me eventueel lid maken om een test user aan te maken kan ik ineens
testen of ik mijn eigen commentaar kan lezen. :)

nelos-34958 A64

thanks

Op do 15 sep. 2022 om 15:41 schreef Nicolas De Smyter <
***@***.***>:

> Goede suggestie.
> Ik neem het mee om dit te bespreken. Dit is niet zomaar iets dat ik zelf
> de beslissing over wil nemen.
> Ik heb ondertussen wel al #67
> <https://github.com/Nelos-VZW/dives-2/issues/67> aangemaakt hiervoor.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Nelos-VZW/dives-2/discussions/60#discussioncomment-3654049>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3CEUVCH67QPUGNHM63NTETV6MRSFANCNFSM6AAAAAAQMNFKKI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
vriendelijke groeten,

michel walgraeve
0477/20/53/32

**Reply by ndsmyter** [reply]
Ik heb je nu toegevoegd. Zal wel pas actief zijn na de volgende migratie morgen (16/09/22).
Zie https://github.com/Nelos-VZW/dives-2/discussions/48#discussioncomment-3642077 voor meer informatie.
